### DISCUSSION TITLE
RCT Discord?

### DISCUSSION THREAD

**Discussion by natetewelde** [lukasbach/react-complex-tree#257] [Ideas]
I recommend a discord for people who use the library. This is a common trend with a lot of libraries, and people will get questions answered much quicker this way. If you need help setting it up, I'd be more than happy to assist.

**Comment by eascan** [comment]
has this been created by chance?

**Comment by downright-development** [comment]
+1 to this!! This is the first thing I looked for :)

**Comment by lukasbach** [comment]
As a quick update on this: I have mentioned a few times in some other issues or discussions, that I have been working on a successor library for react-complex-tree, [Headless Tree](https://headless-tree.lukasbach.com/). I didn't want to make too much noise about this as it was too early to be a viable replacement, in fact I've been working on it for around 2 years now, but it's pretty finalized now and I will release it as stable very soon. I have created a discord community to allow people to chime in on the future of this project, and have also designated this discord as community for react-complex-tree support as well. If anyone is interested in joining, feel free to join the Discord community and help with the future of react-complex-tree and Headless Tree :)

https://discord.gg/KuZ6EezzVw
